syntax = "proto3";

package hello;

service Query {
    rpc queryHello(HelloRequest) returns(HelloReply){}
}

message HelloRequest {
    string name = 1;
}

message HelloReply{
    string name = 1;
}
